package.json

```diff
@@ -32,10 +32,10 @@
     "clean": "backstage-cli package clean"
   },
   "dependencies": {
-    "@backstage/catalog-model": "^1.1.3",
-    "@backstage/core-components": "^0.12.0",
-    "@backstage/core-plugin-api": "^1.1.0",
-    "@backstage/plugin-catalog-react": "^1.2.1",
+    "@backstage/catalog-model": "^1.1.4",
+    "@backstage/core-components": "^0.12.2",
+    "@backstage/core-plugin-api": "^1.2.0",
+    "@backstage/plugin-catalog-react": "^1.2.3",
```

Comment:
https://backstage.github.io/upgrade-helper/?from=1.8.1&to=1.9.1 again: this package was upgraded to `^1.2.2` instead of `^1.2.3`.

Reply:
This seems an issue in the backstage side. The release 1.9.1 didn't update the packages for the upgrade-helper as expected. Check here: https://github.com/backstage/backstage/commit/a3ea732bd5786481af7e5bd0abfcfe94526f7d0f#diff-0779b12cbc8c9735b7a0d85ce26359b759e53faa8d27a0ed01ecc4b45ba3d7be. The package was upgraded here. And it could happen to others as well